contrib/systems.md/bank-account.md

```diff
@@ -82,6 +91,11 @@ mod BANK-ACCOUNT-CTOR is
     op mt       : -> MsgConf [ctor] .
     op _,_ : MsgConf MsgConf -> MsgConf [ctor assoc comm id: mt] .
 
+    op debts : MsgConf -> Nat .
+    ---------------------------
+    eq debts(mt)                 = 0 .
+    eq debts(withdraw(N) , MSGS) = N + debts(MSGS) .
+
```

Comment:
The debts() function is a part of the ctor spec even though it really doesn't behave like a ctor. Is this intentional? If so, it would be good to leave a comment.

Reply:
Alright, well, we can keep the name for now and change it to something better later if it comes up.